src/hint_processor/builtin_hint_processor/hint_utils.rs

```diff
@@ -25,11 +24,11 @@ pub fn insert_value_from_var_name(
 
 //Inserts value into ap
 pub fn insert_value_into_ap(
-    memory: &mut MemoryProxy,
-    run_context: &RunContext,
+    vm_proxy: &mut VMProxy,
     value: impl Into<MaybeRelocatable>,
 ) -> Result<(), VirtualMachineError> {
-    memory.insert_value(&(run_context.get_ap()), value)
+    let ap = vm_proxy.get_ap();
+    vm_proxy.memory.insert_value(&ap, value)
```

Comment:
Just a minor detail, let's avoid creating the `ap` variable:
    `vm_proxy.insert_value(&(vm_proxy.get_ap()), value)`

Reply:
resolved https://github.com/lambdaclass/cairo-rs/pull/448/commits/798f4a71ced8dad6d12be8e25a1c559fa752fdf8